cmd/planner-agent/main.go

```diff
@@ -108,5 +107,19 @@ func (a *agentCmd) readFile(filename string) (string, error) {
 		return string(bytes.TrimSpace(content)), nil
 	}
 
-	return "", errors.New("agent_id not found")
+	return "", fmt.Errorf("file not found: %s", filename)
+}
+
+func (a *agentCmd) readPersistentFile(filename string) (string, error) {
+	filePath := path.Join(a.config.PersistentDataDir, filename)
+	if _, err := os.Stat(filePath); err == nil {
+		content, err := os.ReadFile(filePath)
+		if err != nil {
+			return "", err
+		}
+		return string(bytes.TrimSpace(content)), nil
+	}
+
+	fmt.Println("datadir: ", a.config.DataDir)
```

Comment:
Please use logging instead of `fmt.Println`

Reply:
oops :)